### PR TITLE
Doc Update

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -12,35 +12,38 @@
...
 
-import sphinx_bootstrap_theme
+sys.path.insert(0, os.path.abspath('../..'))
+
 
 
 # -- Project information -----------------------------------------------------
 
-project = 'blenderseed Manual'
+project = 'blenderseed Manaual'
```
Comment (oktomus) [inline docs/conf.py]: Typo here

File: docs/Makefile [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # You can set these variables from the command line.
 SPHINXOPTS    =
 SPHINXBUILD   = sphinx-build
-SPHINXPROJ    = blenderseedManual
+SPHINXPROJ    = blenderseedManaual
```
Comment (oktomus) [inline docs/Makefile]: Typo here

File: docs/conf.py [hunk 3]
```diff
@@ -190,13 +183,16 @@ def setup(app):
...
+    '**': ['localtoc.html', 'searchbox.html'],
+    'using/windows': ['windowssidebar.html', 'searchbox.html'],
+}
 
 
 # -- Options for HTMLHelp output ---------------------------------------------
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'blenderseedManualdoc'
+htmlhelp_basename = 'blenderseedManaualdoc'
```
Comment (oktomus) [inline docs/conf.py]: Typo

**Comment by jdent02** [comment]
Should be fixed.  Yeah I'm not the worlds greatest speller.

File: docs/index.rst [hunk 4]
```diff
@@ -1,20 +1,18 @@
-.. blenderseed Manual documentation master file, created by
-   sphinx-quickstart on Wed Mar 21 22:49:19 2018.
+.. blenderseed Manaual documentation master file, created by
```
Comment (oktomus) [inline docs/index.rst]: Typo

File: docs/make.bat [hunk 5]
```diff
@@ -9,7 +9,7 @@ if "%SPHINXBUILD%" == "" (
 )
 set SOURCEDIR=.
 set BUILDDIR=_build
-set SPHINXPROJ=blenderseedManual
+set SPHINXPROJ=blenderseedManaual
```
Comment (oktomus) [inline docs/make.bat]: Typo

**Comment by jdent02** [comment]
Okay.  Got it now I think.  Who would've thunk one typo during the setup would reproduce in so many places......

**Pull request merged**